app/content/views/strike.py

```diff
@@ -20,6 +26,10 @@ def update(self, request, *args, **kwargs):
         )
 
     def create(self, request):
+        if "enum" in request.data:
+            enum = request.data["enum"]
+            request.data["description"] = get_strike_description(enum)
+            request.data["strike_size"] = get_strike_strike_size(enum)
```

Comment:
Dette burde legges på serializeren

Reply:
Mener det burde legges i viewet ettersom det er data som er sendt i requesten. Slikt burde håndteres i viewet for å holde serializer validation simpelt

Reply:
Godt poeng, er vel ingen enkel metode å gjøre slike ting i serializere heller